app/Party.php

```diff
@@ -705,16 +705,21 @@ public function devices()
 
     public function hoursVolunteered()
     {
-        $lengthOfEventInHours = 3;
-        $extraHostHours = 9;
-        $hoursIfNoVolunteersRecorded = 12;
+        if (!$this->cancelled) {
+            $lengthOfEventInHours = 3;
```

Comment:
Noting to ourselves that we must put a story in about correctly calculating the length of an event for this (knowing that we need to check how this will affect stats everywhere)